src/main/java/de/mediathekview/mserver/crawler/orf/parser/OrfEpisodeDeserializer.java

```diff
@@ -0,0 +1,64 @@
+package de.mediathekview.mserver.crawler.orf.parser;
+
+import com.google.gson.JsonDeserializationContext;
+import com.google.gson.JsonDeserializer;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParseException;
+import com.google.gson.reflect.TypeToken;
+import de.mediathekview.mserver.base.utils.JsonUtils;
+import de.mediathekview.mserver.crawler.orf.OrfEpisodeInfoDTO;
+import de.mediathekview.mserver.crawler.orf.OrfVideoInfoDTO;
+import java.lang.reflect.Type;
+import java.time.Duration;
+import java.util.Optional;
+
+public class OrfEpisodeDeserializer implements JsonDeserializer<Optional<OrfEpisodeInfoDTO>>{
+  
+  private static final String ELEMENT_VIDEO = "video";
+  
+  private static final String ATTRIBUTE_TITLE = "title";
+  private static final String ATTRIBUTE_DESCRIPTION = "description";
+  private static final String ATTRIBUTE_DURATION = "duration";
+  
+  private static final Type OPTIONAL_VIDEOINFO_TYPE_TOKEN = new TypeToken<Optional<OrfVideoInfoDTO>>() {}.getType();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "OPTIONAL_VIDEOINFO_TYPE_TOKEN" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)